packages/rulesets/src/spectral/az-arm.ts

```diff
@@ -211,6 +212,18 @@ const ruleset: any = {
         function: falsy,
       },
     },
+    // RPC Codes: RPC-Get-V1-09, RPC-Arg-V1-01, RPC-Get-V1-06
```

Comment:
Can we link here to a doc elaborating on what are these codes?

Reply:
We do not have a place where these are published yet. The descriptions are currently only available in the ARM OneNote.

Reply:
@bdefoy could you link me to the relevant section in the ARM OneNote? I think we should clarify in the code comments and docs the canonical source of this rules is not public, and thus what is the process of cross-checking/verifying correctness of them. E.g. who to contact if there are any questions about this rule.

Reply:
For all ARM specific rules, you need to contact ARM RPC Governance Team <armrpcgovernanceteam@microsoft.com>